### PR TITLE
(#4575) - Configure CORS on master

### PR DESCRIPTION
**Pull request by daleharvey**

### CODE REVIEW

**Comment by daleharvey** [comment]
Browser tests didnt pass so restarted, but didnt fail immediately, very close to green

File: .travis.yml [hunk 1]
```diff
@@ -37,6 +37,13 @@ before_script:
     # set up CORS
   - "./node_modules/.bin/add-cors-to-couchdb http://127.0.0.1:3000"
 
+    # Configure CORS against master
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/httpd/enable_cors -d '"true"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/origins -d '"*"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/credentials -d '"true"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/methods -d '"GET, PUT, POST, HEAD, DELETE"'
+  - curl -X PUT http://127.0.0.1:3001/_node/node1@127.0.0.1/_config/cors/headers -d '"accept, authorization, content-type, origin, referer, x-csrf-token"'
+
```
Comment (nolanlawson) [inline .travis.yml]: Can you please open a PR on `add-cors-to-couchdb` add this as well? It's been a much-requested feature in that tool for a long time.

**Comment by nolanlawson** [comment]
:+1:, awesome work

**Comment by daleharvey** [comment]
Cheers, merged in https://github.com/pouchdb/pouchdb/commit/c1de0f8e9312f1b2991c3975364953ea47c3570f

**Pull request closed without merging**